src/mesh.rs

```diff
@@ -371,6 +381,25 @@ impl<'a> Primitive<'a> {
         self.json
     }
 
+    /// Returns the bounds (min/max) of the POSITION attribute if there is one, otherwise `None`.
+    /// May panic for invalid glTF files. Use json::validation::Validate::validate_minimally
+    /// to handle this gracefully.
+    pub fn bounds(&self) -> Option<Bounds> {
```

Comment:
That actually makes a lot more sense. How about we make `Bounds` generic instead? So, instead of `fn bounds(&self) -> Bounds` we have `fn position_bounds(&self) -> Bounds<[f32; 3]>`?